synapse/cmds/trigger.py

```diff
@@ -175,6 +176,11 @@ async def _handle_add(self, core, opts):
                     self.printf('Only a single query is allowed')
                     return
                 query = arg
+            elif arg.startswith('"') or arg.startswith("'"):
```

Comment:
To avoid this messy case, maybe we just dont update the cmdr commands to support name fields?

Reply:
:+1: